database/CoreDatabase.java

```diff
@@ -473,7 +479,7 @@ public void delete() {
         }
     }
 
-    static class IsolationManager {
+    public static class IsolationManager {
```

Comment:
Make the class and some of its methods public so that they can be accessed from other packages.